resources/tentative/editors/assets/README.md

```diff
@@ -0,0 +1,2 @@
+longscript.js: the script comes from Apple
```

Comment:
I'm not sure where this comes from actually, but because I'm changing it in a future PR I didn't spend too much time on finding the source.